packages/core/src/checkbox/CheckboxIcon.css

```diff
@@ -24,10 +24,10 @@
   --checkbox-borderWidth: var(--uitk-selectable-borderWidth);
   --checkbox-tick-fill: var(--uitkCheckboxIcon-tick-color, var(--uitk-selectable-primary-foreground-selected));
   --checkbox-viewbox: 14px; /* do not change, matches value in svg viewBox */
-  --checkbox-box-size: calc(var(--checkbox-viewbox) - var(--checkbox-borderWidth));
+  --checkbox-size: calc(var(--checkbox-viewbox) - var(--checkbox-borderWidth));
   --checkbox-box-offset: calc(var(--checkbox-borderWidth) / 2);
 
-  fill: var(--uitkCheckboxIcon-solid-color, var(--uitk-selectable-background));
```

Comment:
Some of these changes don't seem to be only about kebab case?

Reply:
Moved to #346